pivot_controller:
  type: frankpiv_controller/PivotController
  arm_id: $(arg arm_id)
  ee_t_tt_translation: [0, 0, -0.1]
  ee_t_tt_orientation: [0, 0, 0]
  joint_names:
    - $(arg arm_id)_joint1
    - $(arg arm_id)_joint2
    - $(arg arm_id)_joint3
    - $(arg arm_id)_joint4
    - $(arg arm_id)_joint5
    - $(arg arm_id)_joint6
    - $(arg arm_id)_joint7